packages/shared/src/components/cards/v1/RaisedLabel.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+
+export enum RaisedLabelType {
+  Hot = 'Hot',
+  Pinned = 'Pinned',
+  Beta = 'Beta',
+}
+
+const typeToClassName: Record<RaisedLabelType, string> = {
+  [RaisedLabelType.Hot]: 'bg-theme-status-error',
+  [RaisedLabelType.Pinned]: 'bg-theme-bg-bun',
+  [RaisedLabelType.Beta]: 'bg-theme-bg-cabbage',
+};
+
+export interface RaisedLabelProps {
+  type: RaisedLabelType;
+  description?: string;
+  className?: string | undefined;
+  focus?: boolean;
+}
+
+export function RaisedLabel({
+  type = RaisedLabelType.Hot,
+  description,
+  className,
+  focus = false,
+}: RaisedLabelProps): ReactElement {
+  return (
+    <div
+      className={classNames(
+        'absolute right-2 flex flex-row px-2 group-hover:bg-theme-bg-secondary group-focus:-top-0.5 group-focus:bg-theme-bg-secondary',
+        !focus && '-top-[1px] bg-theme-bg-primary',
```

Comment:
This should be available.
```suggestion
        !focus && '-top-px bg-theme-bg-primary',
```

Reply:
Oh, that's how. Thanks, today I learned.